R/dygraph.R

```diff
@@ -70,16 +93,17 @@ dygraph <- function(data, main = NULL, xlab = NULL, ylab = NULL,
   # create x (dygraph attrs + some side data)
   x <- list()
   x$attrs <- attrs
-  x$scale <- periodicity$scale
+  x$scale <- if (format == "date") periodicity$scale else NULL
```

Comment:
I already added a check for that at https://github.com/dgromer/dygraphs/blob/master/inst/htmlwidgets/dygraphs.js#L521. `x.scale` is only used if `jQuery.isNumeric(event.pos)` returns `false`.